app/assets/stylesheets/summary.scss.erb

```diff
@@ -213,4 +213,7 @@ div.vis-network div.vis-navigation {
   }
 }
 
+.no-wrap-text{
```

Comment:
this class already exist in bootstrap `text-nowrap`, see doc for more info https://getbootstrap.com/docs/4.0/utilities/text/#text-wrapping-and-overflow